### PR TITLE
small hours updates

### PR DESCRIPTION
**Pull request by drewvolz**
- Add C-Store (cave) hours
- Remove bag lunch line

### CODE REVIEW

File: data/building-hours/1-4-bag-lunch-line.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 name: Bag Lunch Line
 category: Food
+links:
+  - {title: 'Bag lunch express order', url: 'https://stolaf.cafebonappetit.com/bag-lunch/'}
 
 schedule:
   - title: Hours
-    notes: Closed until further notice
     hours: []
```
Comment (hawkrives) [inline data/building-hours/1-4-bag-lunch-line.yaml]: I think they published pick-up hours, right?

Comment (hawkrives) [reply data/building-hours/1-4-bag-lunch-line.yaml]: "Pick up in Stav between 2-7pm the day before you need your bag lunch."

and

"Bag lunch will take your lunch swipe for the days that you have requested"

**Comment by hawkrives** [comment]
Re-reading the Bag Lunch info on the BonApp website… I propose that we remove Bag Kunch Line from the app. It's no longer a "place with hours"; instead, you have to plan ahead, and there are tradeoffs to using it.

File: data/building-hours/1-4-bag-lunch-line.yaml (hunk 1 above)
Comment (drewvolz) [reply data/building-hours/1-4-bag-lunch-line.yaml]: Right, so do we need hours for walk ups? And then hours for 24 hour in advance order pickup?

Comment (hawkrives) [reply data/building-hours/1-4-bag-lunch-line.yaml]: I vote we delete the bag lunch hours. It feels too specialized now.

**Review by hawkrives — APPROVED** [review]

**Pull request merged**